plugins/woocommerce-admin/client/products/sections/utils.ts

```diff
@@ -81,3 +81,5 @@ export const formatCurrencyDisplayValue = (
 
 	return value === undefined ? value : format( value ).replace( regex, '' );
 };
+
+export function stepUp() {}
```

Comment:
@mdperez86 I assume this can be deleted and was maybe added by accident?

Reply:
Yes, you are right! We don't need this at all. Thanks!!!

Reply:
Solved -> [0c36e0b](https://github.com/woocommerce/woocommerce/pull/35718/commits/0c36e0b19c154b9a67cad525cf4bfd7b2b96fd1e)